app/Migrations/Version20201015123012.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace CultuurNet\UDB3\Silex\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20201015123012 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $schema->dropTable('saved_searches_sapi2');
+    }
+
+    public function down(Schema $schema): void
```

Comment:
I guess we no longer put in the effort to create a `down` migration? I see that was still done in the past but is rather useless.

Reply:
I can see cases where it might be useful, but dropping a read model table is not one of them